scripts/LVGLImage.py

```diff
@@ -872,9 +872,19 @@ def from_png(self,
     def _png_to_indexed(self, cf: ColorFormat, filename: str):
         # convert to palette mode
         auto_cf = cf is None
-        reader = png.Reader(
-            bytes=PngQuant(256 if auto_cf else cf.ncolors).convert(filename))
-        w, h, rows, _ = reader.read()
+        print(filename)
```

Comment:
Let's use `logging.info`
```suggestion
        logging.info(filename)
```